superbit_lensing/shear_profiles/annular_jmac.py

```diff
@@ -266,8 +267,53 @@ def redshift_select(self):
             - Also store the number of galaxies injected into simulation
         '''
 
+        truth_file = self.cat_info['truth_file']
```

Comment:
why are there two of these now?

Reply:
Two of which, I'm sorry?

Reply:
Never mind -- I see it now. That should be removed.